web/modules/webspark/webspark_blocks/webspark_blocks.install

```diff
@@ -297,6 +297,15 @@ function webspark_blocks_update_9028(&$sandbox) {
   _webspark_blocks_update_module_config_list($configs);
 }
 
+/**
+ * Install Layout builder report module
+ */
+function webspark_blocks_update_9029(&$sandbox) {
```

Comment:
Since we're now on D10, let's make this `webspark_blocks_update_10000()`. (Since you're already going to be making a change on this PR. ;) )